django_app/README.md

```diff
@@ -0,0 +1,19 @@
+# How to run and test the django app
+
+## running and testing the django app locally
+
+1. copy `.env.test` to `django_app/.env`
+2. (re)start postgres and minio `docker-compose up -d db minio` 
+3. open you IDE at `django_app` and run the tests here, or navigate to `cd django_app`
+4. the server can be run locally using `poetry run python manage.py runserver`
```

Comment:
I can see the server on http://localhost:8000/ but I'm not seeing static files.

Reply:
I am going to fix this in another PR @brunns